src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/CollectionsShouldImplementGenericInterface.cs

```diff
@@ -88,24 +88,18 @@ public override void Initialize(AnalysisContext analysisContext)
                        interfaceToGenericInterfaceMapBuilder.Add(new KeyValuePair<INamedTypeSymbol, INamedTypeSymbol>(iEnumerableType, genericIEnumerableType));
                    }
 
-                   var userMap = context.Options.GetAdditionalRequiredGenericInterfaces(Rule, context.Compilation, context.CancellationToken);
```

Comment:
We cannot compute option values at compilation level anymore. These option values might differ for each document/folder, hence need to be computed for each symbol/syntax tree.